mars/utils.py

```diff
@@ -428,8 +429,12 @@ def replace_with_fetch_or_copy(n):
                     copied.add(copy.data)
             return node_to_copy[n]
 
-        tileable_graph_builder = TileableGraphBuilder(
-            graph=graph, node_processor=replace_with_fetch_or_copy)
+        if options.optimize_tileable_graph:
```

Comment:
`build_tileable_graph` is used on the client, shall we do the tileable graph optimization on the client? I don't think so.